types/qiniu-js/index.d.ts

```diff
@@ -0,0 +1,371 @@
+// Type definitions for qiniu-js 2.4
+// Project: https://github.com/qiniu/js-sdk#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export as namespace qiniu;
```

Comment:
It seems can be removed

Reply:
I'm afraid not, `qiniu` could be used as global name some time.